src/components/html.cjsx

```diff
@@ -12,3 +12,10 @@ module.exports = React.createClass
       <div className={classes.join(' ')} dangerouslySetInnerHTML={__html:@props.html} />
     else
       <span className={classes.join(' ')} dangerouslySetInnerHTML={__html:@props.html} />
+
+  componentDidMount: ->
```

Comment:
Sure, just added it.

On Fri, Feb 20, 2015 at 1:36 AM, Philip Schatz <notifications@github.com>
wrote:

> In src/components/html.cjsx
> <https://github.com/openstax/tutor-js/pull/23#discussion_r25002014>:
>
> > @@ -12,3 +12,10 @@ module.exports = React.createClass
> >        <div className={classes.join(' ')} dangerouslySetInnerHTML={__html:@props.html} />
> >      else
> >        <span className={classes.join(' ')} dangerouslySetInnerHTML={__html:@props.html} />
> > +
> > +  componentDidMount: ->
>
> Can you include $ = require 'jquery'?
>
> I've had a preference for not including jQuery unless necessary (so far it
> was only used for $.ajax) especially since we are only targeting modern
> browsers. we can chat about it though ; )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openstax/tutor-js/pull/23/files#r25002014>.
>